### PR TITLE
DOC:cliffhanger in special topics 'finite_diff_derivatives.rst'

### LINKED ISSUE #16904: cliffhanger in special topics 'finite_diff_derivatives.rst'
The special topics file [here](https://github.com/sympy/sympy/edit/master/doc/src/special_topics/finite_diff_derivatives.rst) ends with, 

> Next we show how to perform these (and many other discritizations of derivatives) using a
much more efficient approach, credited to Bengt Fornberg, which is incorporated into SymPy.

But there is no more text there. There are presentations in  doc/src/tutorial/calculus.rst and sympy/calculus/finite_diff.py, however, so perhaps only a link to them would be appropriate.

### PR DESCRIPTION
**Pull request by sameshl**
add link to doc/src/tutorial/calculus.rst and sympy/calculus/finite_diff.py in finite_diff_derivatives.rst.

closes #16904

NO ENTRY

### CODE REVIEW

File: doc/src/special_topics/finite_diff_derivatives.rst [hunk 1]
```diff
@@ -249,5 +249,6 @@ given above.
 Next we show how to perform these and many other discritizations of derivatives,  but using a
 much more efficient approach originally due to Bengt Fornberg and now incorporated into SymPy.
 
+:doc:`/src/tutorial/calculus`
```
Comment (smichr) [inline doc/src/special_topics/finite_diff_derivatives.rst]: can this be made to take one directly to `https://github.com/sympy/sympy/blob/master/doc/src/tutorial/calculus.rst#finite-differences`?

Comment (sameshl) [reply doc/src/special_topics/finite_diff_derivatives.rst]: Yes, I could do that

**Comment by smichr** [comment]
Thanks. It's in.

**Comment by sameshl** [comment]
Thank you @smichr. Looking forward to make more contributions to this org! :smiley:

**Pull request merged**